package.json

```diff
@@ -8,31 +8,55 @@
     "build": "babel src -d dist ",
     "eslint": "eslint --ignore-path .gitignore .",
     "dev": "nodemon --exec babel-node ./src/index.js",
```

Comment:
I and the other team members have used dev-start. Change this.

Reply:
Okay adding it